src/components/Sections/SectionsProgress.tsx

```diff
@@ -47,17 +119,8 @@ const SectionsProgress: React.FC<SectionsProgressProps> = ({
             <Grid.Column style={{ minWidth: 100 }} floated="left" width={4}>
               <p>{details.title}</p>
             </Grid.Column>
-            <Grid.Column style={{ width: 200, paddingRight: '5%' }} floated="right" width={4}>
-              {progress && getProgress(progress)}
-            </Grid.Column>
-            <Grid.Column style={{ minWidth: 100, padding: 0 }} width={2}>
-              {firstStrictInvalidPage !== null &&
-                sectionCode === firstStrictInvalidPage.sectionCode && (
-                  <Button color="blue" onClick={() => resumeApplication(firstStrictInvalidPage)}>
-                    {strings.BUTTON_APPLICATION_RESUME}
-                  </Button>
-                )}
-            </Grid.Column>
+            {canEdit && progress && <SectionProgress {...progress} />}
```

Comment:
Hmm, I think we want to show this even if can't edit (when LOQ is presented and before application is restarted

Reply:
Yes, I wasn't sure. But to be improved when we start working on issue [Epic#13 - Application Changes Required](https://github.com/openmsupply/application-manager-web-app/issues/94)